bcgov/HOUS-permit-portal
# Add base layout and navigation structure

[BPHH-2430]

## Description

Adds the base layout and navigation structure for the part 3 checklist form

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

[BPHH-2430](https://hous-bssb.atlassian.net/browse/BPHH-2430)

## Steps to QA

navigate to "/part-3-step-code" (logged in or logged out) - see the basic structure with a left navigation bar to switch between screens. 

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?




[BPHH-2430]: https://hous-bssb.atlassian.net/browse/BPHH-2430?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[BPHH-2430]: https://hous-bssb.atlassian.net/browse/BPHH-2430?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
